### ISSUE TITLE
Notifications for Flutter Web - Feature Request

### ISSUE THREAD

**Issue by adplant** [aws-amplify/amplify-flutter#3103]
I'm sure this is being tracked, but I couldn't find an open issue. Is there any timeline for enabling notifications for Flutter Web?

**Comment by abdallahshaban557** [comment]
Hello @adplant  - we do not have a timeline yet, we are working with the Pinpoint team to enable this feature in the future. We will provide updates on this issue moving forward, you beat me to it! Thank you for reaching out. Please let me know if you have more questions!

**Comment by Lock128** [comment]
It would be good to be able to understand the current dependencies and actions around this...
And of course a timeline ;)

Maybe some hints can help others to collaborate on this?

**Comment by abdallahshaban557** [comment]
Hi @Lock128  - we actually understand the work needed here at this point. We need to enable the use of custom channels with Pinpoint, and then build functionality in the libraries to handle the creation of the endpoint for web and desktop as channels. We are trying to figure out the best way to expose the infrastructure set up pieces for our community.

**Comment by Lock128** [comment]
With "infrastructure" - I assume you mean the required backend-infrastructure (in Amplify or other IaC) to run this piece?

If yes: let's release an "fuill example" application that includes the required backend definitions? ;)

**Comment by abdallahshaban557** [comment]
Yes, that is correct! We are chasing down some open questions we have, and afterwards we will need to figure out the changes needed for the Amplify Flutter library. We will provide updates on this ticket when we have an ETA.

**Comment by KevinHRea** [comment]
For the record this is a feature that we would definitely like to see.
